turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -85,6 +85,7 @@ impl StandardBroadcastRun {
                     keypair,
                     &[],  // entries
                     true, // is_last_in_slot,
+                    None, // chained_merkle_root
```

Comment:
Maintaining last Merkle root here and elsewhere in `turbine/src/broadcast_stage` will be done separately in the follow up commits.